README.md

```diff
@@ -1,17 +1,16 @@
 # gluster-kubernetes
 
-## Dynamic, persistent storage for Kubernetes with GlusterFS
+## GlusterFS + heketi on Kubernetes
```

Comment:
Hm, well, we host none of the components, but this is the project that glues them all together to achieve that broader mission of providing that dynamic, persistent storage for containers, is my understanding. We achieve this by providing the glue code to comfortably deploy the needed resources. I think the proposed headline is not a lot less deceiving than the current one since it does not mention that we only provide means of deploying those, but could make one think that we are actually providing gluster and heketi....